src/py42/exceptions.py

```diff
@@ -166,6 +174,11 @@ def raise_py42_error(raised_error):
         raise Py42ForbiddenError(raised_error)
     elif raised_error.response.status_code == 404:
         raise Py42NotFoundError(raised_error)
+    elif (
```

Comment:
Instead of raising this here, it should be raised during `create_user()`. Otherwise, we are checking every API call

Reply:
This has been changed! Sorry for the churn on style commits.... For some reason the local style linter doesn't work for me!